scripts/config_check/Disabled-Features.txt

```diff
@@ -0,0 +1 @@
+After running the refresh.sh command, this file will contain a list of disabled wolfSSL features.
```

Comment:
My concern is this is a tracked file, so after running the script this show as modified. I think it would be better to not track and have it in the `.gitignore`.

Reply:
interesting feedback; I was specifically interested in "_what changed since I last ran the configure refresh_".

Reply:
Then just save off a .old version.

Reply:
added to `.gitignore`